WooCommerce/Classes/Model/CustomHelpCenterContent.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import WordPressAuthenticator
+
+/// For holding the custom help center content URL
+/// and analytics tracking values
+///
+struct CustomHelpCenterContent {
+    /// Custom help center web page's URL
+    ///
+    let helpCenterContentURL: URL
+
+    /// Provides a dictionary for analytics tracking
+    ///
+    let trackingProperties: [String: String]
+}
+
+extension CustomHelpCenterContent {
+    enum Key: String {
```

Comment:
nit: it might be helpful adding a comment about its raw value being the Tracks event property value

Reply:
Done in 82f9d94ae281c53df3c8f429b48ffedcc2792bfb